spec/views/users_show_spec.rb

```diff
@@ -32,7 +32,7 @@
   end
 
   scenario 'Clicking on a post title redirects to showing more more details about post page' do
-     click_on("Post ##{post.id}")
-     expect(current_path).to eq(user_posts_path(user_id: post.author_id, id: post.id))
-   end
+    click_on("Post ##{post.id}")
+    expect(current_path).to eq(user_posts_path(user_id: post.author_id, id: post.id))
+  end
```

Comment:
- [ ] Great job for adding the test that redirects to the post's show page when you click on a post. but I think you missed a test Tests that asserts you can see a section for pagination if there are more posts than fit on the view. as mentioned by the privous reviewer so kindly try to work on to get your project approve ⭐🔥


- I highly recommend to make each test in seprate `it` block and to wrap them in `describe` block so you get your tests organzied and clean and readable 🔥⭐